crypto/rand/rand_lcl.h

```diff
@@ -198,7 +200,7 @@ struct rand_drbg_st {
     RAND_DRBG_cleanup_entropy_fn cleanup_entropy;
     RAND_DRBG_get_nonce_fn get_nonce;
     RAND_DRBG_cleanup_nonce_fn cleanup_nonce;
-};
+} RAND_DRBG;
```

Comment:
Not sure why you're adding this typedef here, as it's already done in `ossl_typ.h`...  and obviously, that one is included first, or a compiler would probably have complained that the typedef doesn't exist when it hits the `RAND_DRBG *parent;` line...

Reply:
I'm sorry, I put that more harshly than I intended...